### PR TITLE
Create buddy-program.md

### PR DESCRIPTION
**Pull request by mercadon**
Documenting our onboarding buddy program.

### CODE REVIEW

File: handbook/people-ops/onboarding/index.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - [Teammates (based outside US)](../from-graphbook/onboarding_remote_non_us.md)
 - [Engineers](../../engineering/onboarding.md)
 - [Sales team](../../sales/onboarding/index.md)
+- [Onboarding buddy program](../../buddy-program/onboarding.md)
```
Comment (sqs) [inline handbook/people-ops/onboarding/index.md:7]:
```suggestion
- [Onboarding buddy program](../buddy-program.md)
```

File: handbook/people-ops/buddy-program.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+**Frequency**
+- Month 1: 1x per week for 25 min.
+ 
+- Month 2: 1x every two weeks for 25 min.
+
+- Month 3: 1x a month for 25 min.
+
+
+## Buddy selection
```
Comment (dadlerj) [inline handbook/people-ops/buddy-program.md:33]: Anything about similar level of seniority?

Comment (dadlerj) [reply handbook/people-ops/buddy-program.md:33]: Also, how long should someone be on the team before they become eligible to be a buddy themselves? Just once they graduate from their own buddy program?

File: handbook/people-ops/buddy-program.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+**Helpful:** shows them the ropes of the organization and gives them tips on navigating it. 
+
+
+
+## Onboarding buddy logistics
+**Total length**
+
+- 3 months
+
+**Frequency**
```
Comment (dadlerj) [inline handbook/people-ops/buddy-program.md:25]: Curious if there's any new plan to check in or enforce this, or if it should rely on the buddy staying on top of it? Not sure we need anything here, it seems like a self-reinforcing system, but just wondering.

File: handbook/people-ops/buddy-program.md (hunk 2 above)
Comment (mercadon) [reply handbook/people-ops/buddy-program.md:33]: Good points! We've had folks on the exec team be buddies recently and I don't want to restrict anyone by seniority for now– but will keep in mind as we scale! 

I did add a bullet point for eligibility (3 months, so as soon as they've completed their buddy program).

File: handbook/people-ops/buddy-program.md (hunk 3 above)
Comment (mercadon) [reply handbook/people-ops/buddy-program.md:25]: I added a point to make myself responsible for setting up the recurring invites!

**Pull request merged**